packages/just-scripts/src/tasks/sassTask.ts

```diff
@@ -61,7 +60,23 @@ export function sassTask(
                   const css = result.css.toString();
 
                   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-                  postcss([autoprefixerFn, ...postcssPlugins!])
+                  const plugins = [autoprefixerFn, ...postcssPlugins!];
+
+                  // If the rtl plugin exists, insert it after autoprefix.
+                  if (postcssRtl) {
+                    plugins.splice(1, 0, postcssRtl({}));
```

Comment:
```suggestion
                    plugins.splice(plugins.indexOf(autoprefixerFn
                    ), 0, postcssRtl({}));
```

Reply:
it should be + 1 to be after autoprefixer, but I can do that

Reply:
Fixed!